config/secrets.js

```diff
@@ -59,3 +57,17 @@ module.exports.updateFetching = function(flag, cb) {
     return cb(err);
   });
 };
+
+// update settings
+module.exports.update = function(type, settings, cb) {
+  cb = cb || function() {};
+
+  for (var key in settings) {
+    var item = type + ':' + key;
+    nconf.set(item, settings[key]);
+  }
+
+  nconf.save(function(err) {
+    return cb(err);
+  });
+};
```

Comment:
Yes everything should be in the database. The sooner we can get that done
the better. Currently things are still kind of a mess. So many files in the
config folder.

On Mon, Feb 29, 2016 at 11:05 PM, Andrés Moreno <notifications@github.com>
wrote:

> In config/secrets.js
> <https://github.com/TID-Lab/aggie/pull/147#discussion_r54520462>:
>
> > @@ -59,3 +57,17 @@ module.exports.updateFetching = function(flag, cb) {
> >      return cb(err);
> >    });
> >  };
> > +
> > +// update settings
> > +module.exports.update = function(type, settings, cb) {
> > +  cb = cb || function() {};
> > +
> > +  for (var key in settings) {
> > +    var item = type + ':' + key;
> > +    nconf.set(item, settings[key]);
> > +  }
> > +
> > +  nconf.save(function(err) {
> > +    return cb(err);
> > +  });
> > +};
>
> Yes, it is temporary.
>
> Toggling fetching also depends on this. We may want to move that setting
> to the secrets.json and eventually to the database as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/TID-Lab/aggie/pull/147/files#r54520462>.
>



-- 
Tom Smyth

Worker-Owner, Sassafras Tech Collective
Specializing in innovative, usable tech for social change
sassafras.coop *·* @sassafrastech

Resident, Touchstone Cohousing
touchstonecohousing.org